src/common/imagpng.cpp

```diff
@@ -323,6 +384,18 @@ wxPNGHandler::LoadFile(wxImage *image,
     }
 
     png_read_image( png_ptr, lines );
+
+    // custom options
+    for (int textIndex = 0; textIndex < info_ptr->num_text; ++textIndex)
+    {
+        if (info_ptr->text[textIndex].compression == PNG_TEXT_COMPRESSION_NONE)
+        {
+            wxString key = info_ptr->text[textIndex].key;
+            wxString value = wxString(info_ptr->text[textIndex].text, info_ptr->text[textIndex].text_length);
```

Comment:
This is wrong: we don't want to use the current locale encoding (which can be anything at all) when decoding PNG data. It probably should be `wxString::FromUTF8()`, but I'm not sure if PNG properties are always guaranteed to be in UTF-8. If not, we may need to fallback on `From8BitData()`.